src/app/pages/signing/image-modal/image-modal.component.ts

```diff
@@ -12,15 +8,16 @@ import { ErrorAlertComponent } from '../error-alert/error-alert.component';
   styleUrls: ['./image-modal.component.css'],
 })
 export class ImageModalComponent {
-  modalRef: BsModalRef;
+  static maxFileSize = 8 * 1000000; // 8 MB in bytes
```

Comment:
I'm using `8 * 1024 * 1024` in `storage.rules`. We should make these match
```suggestion
  static maxFileSize = 8 * 1024 * 1024; // 8 MiB in bytes
```